src/syrk.cc

```diff
@@ -59,8 +59,8 @@ void syrk(
     std::vector<uint8_t>  gemm_vector(A.nt());
     uint8_t* bcast = bcast_vector.data();
     uint8_t* gemm  =  gemm_vector.data();
-    const int default_priority = 0;
-    const int default_queue = 0;
+    const int priority_0 = 0;
```

Comment:
Here the constant declarations are not gather at the top. Is it okay for now?

Reply:
Fixed.